style.css

```diff
@@ -105,6 +114,12 @@ hr {
   max-width: 450px !important;
 }
 
+#driver-popover-item * {
```

Comment:
I've opened [this issue](https://github.com/CodeWithAloha/Hawaii-Zoning-Atlas/issues/114) to eventually clean this up, once we get off of our older version of `driver.js`. Related: https://driverjs.com/docs/styling-popover